### PR TITLE
[Add]create parallel-meta-suite recipe

### PR DESCRIPTION
**Pull request by qdu-bioinfo**
[Add]create parallel-meta-suite recipe

### CODE REVIEW

**Comment by qdu-bioinfo** [comment]
@epruesse I'm sorry to disturb you, but I have a problem here. While Circle CI test my recipe, I got a `g++: No such file or directory`. But I tested this recipe on my own PC with conda-build, it works well. Could you tell me what to do to solve this problem?

**Comment by qdu-bioinfo** [comment]
@marcelm Sorry to disturb you. I got a problem here. While Circle CI testing my recipe, I got a `g++: No such file or directory`. It seems that the `src` folder can not be found. But I tested this recipe on my own PC with conda-build, it works well. Could you tell me how to solve this problem?

**Comment by marcelm** [comment]
I was also able to build the package locally. The problem is that `make` hardcodes `g++` for compilation, which will not use the compiler provided by Conda, but the one installed on your system. This is one of the lines that is printed during the build:
```
g++ -c src/ExtractRNA.cpp -o bin/ExtractRNA.o -Wno-deprecated
```
It should actually use whichever compiler Conda has configured in the `$CXX` environment variable. This can often be achieved by calling make like `make CXX=${CXX}`.

**Comment by marcelm** [comment]
Looking at the Makefile, I think you need `make CC=$CXX`

**Comment by qdu-bioinfo** [comment]
Thanks a million. I have tried `make CC=$CXX`. It works! But this time I can not pass the MacOS test... The problem is clang compiler does not have `OpenMP` library. I tried to add `OpenMP`  and `llvm-OpenMp` in the `requirements` section, it didn't work. What should I do?

**Comment by marcelm** [comment]
I don’t know how to fix macOS builds, sorry. If supporting only Linux is good enough for you, you can make your build section look like this to skip the macOS build:
```
build:
  number: 0
  skip: True  # [osx]
```
Otherwise, you may want to ask on the [chat](https://gitter.im/bioconda/Lobby#) otherwise.

**Comment by qdu-bioinfo** [comment]
> I don’t know how to fix macOS builds, sorry. If supporting only Linux is good enough for you, you can make your build section look like this to skip the macOS build:
> 
> ```
> build:
>   number: 0
>   skip: True  # [osx]
> ```
> 
> Otherwise, you may want to ask on the [chat](https://gitter.im/bioconda/Lobby#) otherwise.

I will skip macOS and figure this one myself! I truly appreciate your help for the make command!

File: recipes/parallel-meta-suite/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+make CC=$CXX 
+
+cp -r bin/* ${PREFIX}/bin
+
+# # path for database and example
+ParallelMETA_path=${PREFIX}/${PKG_NAME}-${PKG_VERSION}
+mkdir -p ${ParallelMETA_path}
+
+# set VIBRANT DB PATH variable on env activation
```
Comment (marcelm) [inline recipes/parallel-meta-suite/build.sh:12]: This comment isn’t quite correct, or is it? You can just delete that line, it’s clear what an activate/deactivate script does.

**Review by marcelm — APPROVED** [review]

File: recipes/parallel-meta-suite/build.sh (hunk 1 above)
Comment (qdu-bioinfo) [reply recipes/parallel-meta-suite/build.sh:12]: The comment is correct. The database which we used is very big, more than 700MB. I don't think add this to the `source` is a good choice. So I define a path in the activate script, the download command will download the database in this folder.

Comment (marcelm) [reply recipes/parallel-meta-suite/build.sh:12]: I just found it confusing that the comment seems to imply that a variable named `VIBRANT_DB_PATH` or similar would be set, but the actual variable name is `ParallelMETA_path`

Comment (marcelm) [reply recipes/parallel-meta-suite/build.sh:12]: I’ll just merge it

**Comment by marcelm** [comment]
@biocondabot please merge

**Pull request merged**